### DISCUSSION TITLE
pre-built presets

### DISCUSSION THREAD

**Discussion by Taza53** [wwmm/easyeffects#1033] [Ideas]
An optional set of presets that can be downloaded.
Image form another project as an example. ![image](https://user-images.githubusercontent.com/34032665/126066187-196d057c-53c0-4ee5-ac8b-2923663df0af.png)

Q: why not use the lyrebird?
A: It's only pulseaudio

Q: It is very easy to make your own presets?
A: yes, but not good ones.

**Comment by wwmm** [comment]
The main difficulty is creating the "good presets". I don't really have the knowledge to create general purpose presets that could be good for many different people. Is Lyrebird using the rubberband library? If yes I could try to see how they are setting the parameters in each preset.

**Reply by Taza53** [reply]
no i dont think so but presets are defined [https://github.com/lyrebird-voice-changer/lyrebird/blob/master/app/core/presets.py](url) starting from line 63. 
Having a way for the community to contribute presets could be a good idea too.

**Reply by Mlocik97** [reply]
There already are presets here: https://github.com/wwmm/easyeffects/wiki/Community-Presets

And I don't think it would be good to have them as part of EE.

**Reply by Taza53** [reply]
Okay, community presets as a part of EE directly is not a very good idea, but something like github.com/wwmm/easyeffects-comunitypresets. But a couple presets built-in would be helpful, especially for new users. Thanks for the link.

**Reply by Mlocik97** [reply]
I think how it's right now, it's fine... creating repo for community presets is really bad idea, because it would require giving permission to everyone for committing to repo. And it would also require maintaining that is not really possible.
Also preset structure change some time, and all of presets in repo would be invalid, so original author would need to maintain it in "foreign" repo, and creating Pull Request per every change is impractical.

**Reply by p-chan5** [reply]
I also agree with it. Supporting a new `.js` configuration file structure requires recreating the preset from scratch (at least for that version).

**Reply by Lenzelot** [reply]
Hi, new user here. Is it normal that the Community Presets tab is empty? I installed packages `lsp-plugins` and `easyeffects` on arch. What does the user have to see entries in this list?

<details><summary>Community Presets tab screenshot</summary>
<p>
<img width="1254" height="987" alt="Screenshot_20260119_132947" src="https://github.com/user-attachments/assets/79c519e9-a28c-4aed-b995-b43c5976dc9b" />
</p>
</details>

**Reply by wwmm** [reply]
>Is it normal that the Community Presets tab is empty? 

In most cases it is. Community presets will only be available if you install preset packages provided by the community. Something like this one https://aur.archlinux.org/packages/easyeffects-digitalone1-presets-git. THese are different fro mthe packages that provide plugins like `lsp-plugins`.

At this moment community preset packages are not very common.
